tests/lora/utils.py

```diff
@@ -244,27 +250,35 @@ def test_simple_inference_with_text_lora(self):
                         check_if_lora_correctly_set(pipe.text_encoder_2), "Lora not correctly set in text encoder 2"
                     )
 
-            output_lora = pipe(**inputs, generator=torch.manual_seed(0)).images
+            output_lora = pipe(**inputs, generator=torch.manual_seed(0))[0]
             self.assertTrue(
-                not np.allclose(output_lora, output_no_lora, atol=1e-3, rtol=1e-3), "Lora should change the output"
+                not np.allclose(output_lora, output_no_lora, atol=expected_atol, rtol=expected_rtol),
+                "Lora should change the output",
             )
 
-    def test_simple_inference_with_text_lora_and_scale(self):
+    def test_simple_inference_with_text_lora_and_scale(self, expected_atol: float = 1e-3, expected_rtol: float = 1e-3):
         """
         Tests a simple inference with lora attached on the text encoder + scale argument
         and makes sure it works as expected
         """
         scheduler_classes = (
             [FlowMatchEulerDiscreteScheduler] if self.uses_flow_matching else [DDIMScheduler, LCMScheduler]
         )
+        call_signature_keys = inspect.signature(self.pipeline_class.__call__).parameters.keys()
```

Comment:
We needed to modify these at a few places to make sure the correct attention kwargs argument was passed. IMO, we should have the same variable naming convention for all pipelines (perhaps, something for the 1.0.0 refactor) - `attention_kwargs`